jest.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    testEnvironment: 'jsdom', // Needed for React Testing Library
+    setupFilesAfterEnv: ['@testing-library/jest-dom'],
+  };  
```

Comment:
Can the jest config be added to package.json as well? https://jestjs.io/docs/configuration

I felt it might simply the number of files we have in the repo. But some believe in other way as well

Reply:
I agree with your approach! I didn't realise I could do that, added to package.json